fixtures/iso/INDEX

```diff
@@ -41,9 +41,17 @@ embed-areas-2023-03.x86_64.iso.xz
         live-initrd-network
     installer-config-directives from 0.16.0
 
-embed-areas-2023-03.s390x.iso.xz
```

Comment:
Why is this one being removed?  We should always retain old fixtures; that's the purpose of having them.